pkg/core/terraform/tf.go

```diff
@@ -155,33 +157,70 @@ func (sw *StdWriter) GetString() string {
 	return s
 }
 
-func (tf Terraform) formatTerraformWorkspaces(list string) (string) {
+func (tf Terraform) formatTerraformWorkspaces(list string) string {
 
 	list = strings.TrimSpace(list)
 	char_replace := strings.NewReplacer("*", "", "\n", ",", " ", "")
 	list = char_replace.Replace(list)
 	return list
 }
 
+func copyGitHubVars() (map[string]string, error) {
+	githubVars := os.Getenv("GITHUB_VARS")
```

Comment:
there shouldn't be anything provider specific inside of "core"

Reply:
yeah probably I need to move it a few levels up